tasks/patches/add-sbom-and-push.yaml

```diff
@@ -126,9 +126,9 @@
     imagePullPolicy: Always
     env:
       - name: QUARKUS_REST_CLIENT_CACHE_SERVICE_URL
-        value: "http://hacbs-jvm-cache.jvm-build-service.svc.cluster.local"
+        value: "http://jvm-build-workspace-artifact-cache.$(context.taskRun.namespace).svc.cluster.local"
```

Comment:
ah @psturc, re: our test discussion earlier today, yes, I *knew* wokspace based caches were on the horizon

I don't *think* this change will have a direct bearing on the component / hacbs tekton bundle test case we discussed, assuming there are no functional issues when we pivot to it, but certainly follow this PR/story, and as we discussed, we should have a common conversation with @stuartwdouglas at some point around making sure the build is accessing caches/proxies as desired.